convertir '' en 0 si el tipo es `int` en administrar_general

Ejemplo del error:
```
web_1    | [INFO] (2019-11-08 17:42:57,742 views) modifico 85 objetos tipo <class 'materias.models.Turno'>                                                                                                                                                    
web_1    | [ERROR] (2019-11-08 17:42:57,840 log) Internal Server Error: /materias/administrar_alumnos/2020/S                                                                                                                              
web_1    | Traceback (most recent call last):                                                                                                                                                                                                                
web_1    |   File "/usr/local/lib/python3.7/site-packages/django/core/handlers/exception.py", line 34, in inner                                                                                                                                                          
web_1    |     response = get_response(request)                                                                                                                                                                                                             
web_1    |   File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 126, in _get_response                                                                                                                        
web_1    |     response = self.process_exception_by_middleware(e, request)                                                                                                                                                                                              
web_1    |   File "/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py", line 124, in _get_response                                                                                                                                        
web_1    |     response = wrapped_callback(request, *callback_args, **callback_kwargs)                                                                                                                                                                                    
web_1    |   File "/usr/local/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
web_1    |     return view_func(request, *args, **kwargs)
web_1    |   File "/usr/local/lib/python3.7/site-packages/django/contrib/auth/decorators.py", line 21, in _wrapped_view
web_1    |     return view_func(request, *args, **kwargs)
web_1    |   File "/codigo/distribucion/materias/views.py", line 207, in administrar_alumnos
web_1    |     'materias/administrar_alumnos.html', seccion='materias')
web_1    |   File "/codigo/distribucion/materias/views.py", line 176, in administrar_general
web_1    |     v = _type(request.POST[page_field_objeto])
web_1    | ValueError: invalid literal for int() with base 10: ''
```

--- a/materias/tests.py
+++ b/materias/tests.py
@@ -389,6 +389,30 @@ class TestPaginas(TestCase):
         self.assertEqual(nhorario112.aula, 'xyz')
         self.assertEqual(nhorario112.pabellon, '0')
 
+    def test_administrar_converte_vacio_a_0_en_int(self):
+        self.client.login(username='autorizado', password='1234')
+
+        model_to_ktf = {Turno: {'alumnos': ('alumnos', int)},
+                        Horario: {'aula': ('aula', str),
+                                  'pabellon': ('pabellon', int)}}
+        post = dict()
+        for model, key_to_field in model_to_ktf.items():
+            for k_field, (t_attr, _) in key_to_field.items():
+                for o in model.objects.all():
+                    post[f'{k_field}_{o.id}'] = getattr(o, t_attr)
+        # primero pasamos valor vacío, tiene que quedar en 0
+        post[f'alumnos_{self.turno12.id}'] = ''
+        post['cambiar'] = True
+        url = f'/materias/administrar_alumnos/{self.anno}/{self.cuatrimestre.name}'
+        self.client.post(url, post)
+        self.assertEqual(self.turno12.alumnos, 0)
+        # ahora pasamos valor no parseable, tiene que tirar excepción
+        # Eso resulta en un 500 Internal Server Error. No está bueno pero sería raro que llegue algo
+        # que no es número ni es vacío hasta ahí.
+        post[f'alumnos_{self.turno12.id}'] = 'x'
+        with self.assertRaises(ValueError):
+            self.client.post(url, post)
+
     def test_administrar_cargas_docentes(self):
         self.client.login(username='autorizado', password='1234')
         self._agrega_docentes()

--- a/materias/views.py
+++ b/materias/views.py
@@ -164,7 +164,7 @@ def administrar_general(request, anno, cuatrimestre, key_to_field, url, seccion=
                     objetos = Turno.objects.filter(anno=anno, cuatrimestre=cuatrimestre)
                 elif  modelo == Horario:
                     objetos = Horario.objects.filter(turno__anno=anno, turno__cuatrimestre=cuatrimestre)
-                logger.info('modifico %d objetos tipo %s', objetos.count(), modelo)
+                logger.info('modifico %d objetos tipo %s', objetos.count(), modelo.__class__.__name__)
 
                 for objeto in objetos:
                     for page_field, (field, _type) in modelo_key_to_field.items():
@@ -173,7 +173,16 @@ def administrar_general(request, anno, cuatrimestre, key_to_field, url, seccion=
                             # checkbox aparece solo si está marcado
                             v = page_field_objeto in request.POST
                         else:
-                            v = _type(request.POST[page_field_objeto])
+                            v_post = request.POST[page_field_objeto]
+                            try:
+                                v = _type(v_post)
+                            except ValueError:
+                                if _type is int and v_post == '':
+                                    v = 0
+                                else:
+                                    logger.exception('no pude convertir "%s" a tipo %s para el input %s',
+                                                     v_post, _type.__name__, page_field_objeto)
+                                    raise
                         setattr(objeto, field, v)
                         logger.debug('cambiando %s a obj. %s por %s', page_field, objeto, v)
                     objeto.save()

